tools/depth-quality/depth-quality-model.cpp

```diff
@@ -865,20 +886,29 @@ namespace rs2
                         continue;
                     }
 
-                    auto stream_type = depth_frame.get_profile().stream_type();
+                    auto profile = depth_frame.get_profile();
+                    auto stream_type = profile.stream_type();
 
                     if (RS2_STREAM_DEPTH == stream_type)
                     {
                         float su = 0, baseline = -1.f;
-                        rs2_intrinsics intrin;
-                        int gt_mm;
-                        bool plane_fit_set;
-                        region_of_interest roi;
+                        rs2_intrinsics intrin{};
```

Comment:
[Don't introduce a variable (or constant) before you need to use it](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es21-dont-introduce-a-variable-or-constant-before-you-need-to-use-it)!
Also, you only use this variable to update `_depth_intrinsic`, so it is redundant altogether...

Reply:
The '_depth_intrinsic' is intended to be used in the main thread, therefore a local copy is created.
Note that the assignment is mutex-protected